ext/mbstring/libmbfl/filters/mbfilter_big5.c

```diff
@@ -237,11 +237,6 @@ int mbfl_filt_conv_big5_wchar(int c, mbfl_convert_filter *filter)
 		} else {
 			CK((*filter->output_function)(MBFL_BAD_INPUT, filter->data));
 		}
-		break;
-
-	default:
-		filter->status = 0;
-		break;
```

Comment:
Impossible defaults should be replaced with `EMPTY_SWITCH_DEFAULT_CASE()` to assert that.

Reply:
Thanks, just pushing an update.